rpm/libpcap.spec

```diff
@@ -0,0 +1,77 @@
+Name: libpcap
+Version: 1.5.3
+Release: 1
+Summary: A system-independent interface for user-level packet capture
+Group: Development/Libraries
+License: BSD with advertising
```

Comment:
I wonder what this advertising here is, the license file to me seems generic BSD.

https://fedoraproject.org/wiki/Licensing:BSD?rd=Licensing/BSD#BSDwithAdvertising